database/api/src/main/java/com/neueda/jetbrains/plugin/graphdb/database/api/query/GraphQueryResult.java

```diff
@@ -20,4 +20,10 @@
     List<GraphRelationship> getRelationships();
 
     List<GraphQueryNotification> getNotifications();
+
+    boolean hasPlan();
+
+    boolean hasProfile();
+
+    GraphQueryPlan getPlan();
```

Comment:
As I understand "hasPlan()" is main method. And it is accompanied with "getPlan()"

In this case "hasProfile" is misleading. I would prefer it be called "isProfilePlan()" to clearly define that it actually specifies type of plan.